<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuClass</class>
 <widget class="QMainWindow" name="MenuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>802</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>597</width>
    <height>802</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>597</width>
    <height>802</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/itsmygo.png</normaloff>images/itsmygo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="appTitle">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>51</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Minecraft</string>
    </property>
   </widget>
   <widget class="QPushButton" name="simpleModeBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>单人模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="duoModeBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>双人模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>读取存档</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>640</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playControlBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="appTitle2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>44</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Link</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitBtn</sender>
   <signal>clicked()</signal>
   <receiver>MenuClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>playControlSlot()</slot>
 </slots>
</ui>
